features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/details/RoomMemberDetailsPresenter.kt

```diff
@@ -53,8 +54,13 @@ class RoomMemberDetailsPresenter @AssistedInject constructor(
         var confirmationDialog by remember { mutableStateOf<ConfirmationDialog?>(null) }
         val roomMember by room.getRoomMemberAsState(roomMemberId)
         // the room member is not really live...
-        val isBlocked = remember {
-            mutableStateOf(roomMember?.isIgnored.orFalse())
+        val isBlocked: MutableState<Async<Boolean>> = remember(roomMember) {
```

Comment:
are "blocked" and "ignored" synonyms in this context?

Reply:
Yes, I admit we should only use one term to avoid confusion. Technically, the user will be [ignored](https://spec.matrix.org/latest/client-server-api/#ignoring-users), from the product perspective, the user will be `blocked`.